docs/design/include/mcmini/coordinator/model_to_system_map.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include "mcmini/forwards.hpp"
+#include "mcmini/misc/optional.hpp"
+#include "mcmini/model/state.hpp"
+
+namespace mcmini {
```

Comment:
Why do we have a mcmini namespace?
Will some of this code have a different namespace?  If we're executing here, it is 'mcmini'.  why make everybody do twice as much typing every time they use a variable in GDB?

Reply:
I can remove the extraneous `mcmini` namespace as you're right it's going to wrap everything anyway